compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -818,15 +819,14 @@ execMergeDars darFp1 darFp2 mbOutFp = do
                     ]
         m1 <- getEntry mfPath dar1
         let m' = do
-                l <- lines $ BSC.unpack $ BSL.toStrict $ ZipArchive.fromEntry m1
-                pure $
-                    maybe
-                        l
-                        (const $
-                         breakAt72Chars $
-                         "Dalfs: " <> intercalate ", " dalfNames)
-                        (stripPrefix "Dalfs:" l)
-        pure $ ZipArchive.toEntry mfPath 0 $ BSL.fromStrict $ BSC.pack $ unlines m'
+                l <- BSC.lines $ ZipArchive.fromEntry m1
+                -- TODO This should use the proper manifest reader.
+                -- At the moment this relies on the fact that the input manifest
+                -- does not have Dalfs: entries that are split over multiple lines.
```

Comment:
Do we actually still produce DARs that satisfy this assumption?

Reply:
I have no idea. Presumably @robin-da tested the merge command at some point :slightly_smiling_face: